README.md

```diff
@@ -109,6 +110,9 @@ Please refer to the appropriate Cordova documentation for details:
 
 # <a name="build-run-locally"></a>Build and run locally
 
+**Make sure you have `CORDOVA_ANDROID_GRADLE_DISTRIBUTION_URL` in your system path** 
+>`CORDOVA_ANDROID_GRADLE_DISTRIBUTION_URL=https://services.gradle.org/distributions/gradle-4.10.1-all.zip` .

```

Comment:
Maybe mark this a work around rather than a must. It's a bug in cordova-android that we will look into upgrading later down the road

Reply:
Done